Coq.sublime-syntax

```diff
@@ -44,14 +44,14 @@ contexts:
   # Illegal syntax
   illegal_punct:
```

Comment:
Please add a comment explaining why these are disabled. Or, maybe the entire idea of highlighting illegal syntax is unsalvageable? Then we should just get rid of these contexts.

Reply:
Anything in here
https://gitlab.mpi-sws.org/iris/iris
for example, is filled with invalid scopes.
It's not just the unicode (which is now also fixed).

Reply:
Let's just get rid of them then.